briskedu/brisk-learning
# Support extension in Google Classroom assignments

Summary:
Adding support for the extension to Google Classroom assigments. It's basically the same except we get the URL from the iframe once it loads.

https://app.asana.com/0/1203963408726580/1203963408926857

Test Plan:
1. Confirm I can load classroom documents
2. Confirm I can check for plagiarism
3. Confirm placement of elements is same on Google Docs page
4. Confirm everything works the same on standalone documents

<img width="991" alt="classroom" src="https://user-images.githubusercontent.com/15060005/223507602-71e78da1-0c40-4fb3-bd61-4a1ae92a9240.png">
